lib/Backends/Interpreter/InterpreterDeviceManager.cpp

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "InterpreterDeviceManager.h"
+
+using namespace glow;
+using namespace glow::runtime;
+
+namespace glow {
+DeviceManager *createInterpreterDeviceManager(llvm::StringRef name) {
+  return new InterpreterDeviceManager(name);
+}
+} // namespace glow
```

Comment:
you can close this namespace at the very end of the file (InterpreterDeviceManager is also in the glow namespace).

Reply:
Thats true, is it better to do it that way?

Reply:
that way you'll kill using namespace glow above ;)